File: app/src/main/java/com/wolcano/musicplayer/music/utils/ToastUtils.java
package com.wolcano.musicplayer.music.utils;

import android.content.Context;
import android.widget.Toast;

public class ToastUtils {

    private static Toast toastMsg;

    public static void show(Context context, String str) {
        if (toastMsg == null) {
            toastMsg = Toast.makeText(context, str, Toast.LENGTH_SHORT);
        } else {
            toastMsg.setText(str);
        }
        toastMsg.show();
    }

    public static void show(Context context, int id) {
        show(context, context.getString(id));
    }

}

File: app/src/main/java/com/wolcano/musicplayer/music/utils/ToastUtils.kt
package com.wolcano.musicplayer.music.utils

import android.annotation.SuppressLint
import android.content.Context
import android.widget.Toast

object ToastUtils {

    private var toastMsg: Toast? = null

    @SuppressLint("ShowToast")
    fun show(context: Context?, str: String?) {
        if (toastMsg == null) {
            toastMsg = Toast.makeText(context, str, Toast.LENGTH_SHORT)
        } else {
            toastMsg!!.setText(str)
        }
        toastMsg?.show()
    }

    fun show(context: Context, id: Int) {
        show(context, context.getString(id))
    }

}